src/char/char.c

```diff
@@ -1488,6 +1489,29 @@ int char_rename_char_sql(struct char_session_data *sd, int char_id)
 	if( char_dat.rename == 0 )
 		return 1;
 
+	if (char_aegis_rename) {
+		int party_id = 0;
+		int guild_id = 0;
+		char *data;
+
+		if (SQL_SUCCESS != SQL->Query(inter->sql_handle, "SELECT `party_id`, `guild_id` FROM `%s` WHERE `char_id`='%d'", char_db, char_id)
```

Comment:
Is this query necessary? Is the info fetched into `char_dat.guild_id` and `char_dat.party_id` not enough?